packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -728,6 +728,13 @@ export class SDKClient {
         throw e;
       }
 
+      // Log the target node account ID, right now, it's populated only for MaxAttemptsOrTimeout error
+      if (e?.nodeAccountId) {
```

Comment:
This is okay given the beta version and end of sprint but I'd prefer we move this logic into `SDKError` and add the node to the existing error logs

Reply:
@nickeynikolovv FYI